Directory.Build.targets

```diff
@@ -1,4 +1,4 @@
-<Project InitialTargets="FindUnity">
+<Project>
```

Comment:
Just for clarification, what's the motivation to remove FindUnity from InitialTargets?